daemon/control.proto

```diff
@@ -294,6 +299,17 @@ message DaemonToController {
 	optional LogMessage log_message = 12;				// log message received because of OBSERVE_LOG_START
 
 	optional Response response = 13;
+
+	optional uint64 disk_system = 20;		// storage usage in DEVICE_STATS response
+	optional uint64 disk_system_free = 21;		// storage usage in DEVICE_STATS response
+	optional uint64 disk_system_used = 22;		// storage usage in DEVICE_STATS response
+	optional uint64 disk_containers = 23;		// storage usage in DEVICE_STATS response
+	optional uint64 disk_containers_free = 24;	// storage usage in DEVICE_STATS response
+	optional uint64 disk_containers_used = 25;	// storage usage in DEVICE_STATS response
+	optional uint64 mem_total = 26;			// mem utilization in DEVICE_STATS response
+	optional uint64 mem_free = 27;			// mem utilization in DEVICE_STATS response
+	optional uint64 mem_available = 28;		// mem utilization in DEVICE_STATS response
+
```

Comment:
may this should be wrapped in DeviceStats message

Reply:
done